src/types/index.ts

```diff
@@ -113,6 +113,7 @@ export type AnonymizedValue = Value | AnonymizedAggregate;
 export type AnonymizedAggregate = {
   realValue: number;
   anonValue: number | null;
+  aggregateName: string;
```

Comment:
I decided to add this, to not rely only on `typeof v === 'object'` given we now have 2 different kinds of objects populating the rows